docs/editing_guidelines.md

```diff
@@ -0,0 +1,138 @@
+# Guidelines for CL editors
+
+## Naming terms
+
+### General rules
+1. All names should be singular nouns.
+2. Names should not be capitalised, unless they are or contain a proper name (e.g. _Johnston's organ_).
+3. Avoid special characters: use only alpha-numeric characters, space, dash (`-`), slash (`/`), and apostrophe (`'`).
+
+### Advice
+Bear in mind that users will often encounter terms in isolation. Long descriptive names (within reason!) are therefore preferable, especially where there is obvious potential for confusion. For example, _calyx_ (which simply means “cup”) could refer to a structure in the oviduct or in the mushroom body, depending on your field of specialisation. It is therefore better to use _mushroom body calyx_ and _oviduct calyx_ rather than simply _calyx_ alone.
```

Comment:
```suggestion
Bear in mind that users will often encounter terms in isolation. Long, descriptive labels (within reason!) are therefore preferable, especially where there is obvious potential for confusion. For example, _calyx_ (which simply means “cup”) could refer to a structure in the oviduct or in the mushroom body, depending on your field of specialization. It is therefore better to use _mushroom body calyx_ or _oviduct calyx_ rather than simply _calyx_ alone.
```